htdocs/index.html

```
@@ -1 +1,11 @@
-<h3>Hello World</h3>
\ No newline at end of file
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)